src/commands/instance/logs.ts

```diff
@@ -0,0 +1,79 @@
+import { Command, flags } from '@oclif/command'
+import * as inquirer from 'inquirer'
+import execa from 'execa'
+
+export class InstanceLogs extends Command {
+
+  static flags = {
+    instance: flags.string({
+      char: 'i',
+      description: 'Kuzzle instance name',
+    }),
+    follow: flags.boolean({
+      char: 'f',
+      description: 'Follow log output',
+    }),
+  }
+
+  async run() {
+    const { flags } = this.parse(InstanceLogs)
+    let instance: string = flags.instance!;
+    let followOption: boolean = flags.follow
+
+    if (!instance) {
+      const instancesList = await this.getInstancesList()
+
+      let responses: any = await inquirer.prompt([{
+        name: 'instance',
+        message: 'On which kuzzle instance do you want to see the logs',
+        type: 'list',
+        choices: instancesList,
+      }])
+      instance = responses.instance!
+    }
+    try {
+      await this.showInstanceLogs(instance, followOption)
+    } catch {
```

Comment:
Should be on the line below according to our standards
Same remark for others try/catch